.github/workflows/reusable_provenance.yaml

```diff
@@ -0,0 +1,171 @@
+# Builds an artifact from the input build config, and uploads it to Ent. Then generates the SLSA provenance for it and uploads that to Ent too.
+# Returns (1) the name and digest of the generated artifact as a based64-encoded string, and (2) the digest of the provenance.
+name: Build Provenance
+
+on:
+  workflow_call:
+    inputs:
+      build-config-path:
+        required: true
+        type: string
+    # Map the workflow outputs to job outputs
+    outputs:
+      artifact_digest:
+        description:
+          'Base64-encoded digest of the artifact, including its name.'
```

Comment:
not sure what format this is exactly, do you mean the binary name plus a space plus the sha256 digest?